### PR TITLE
Update first-paragraph.mdx

### PR DESCRIPTION
**Pull request by ridetoruin**
Added back ticks.
Added references.
Updated MLA.
Added Authors.

Thank you for wanting to contribute to sPhil! 🧙 🦉

We would like to keep our code and writing as neat and tidy as possible, and would appreciate if you could verify the following. Each PR should _either_ contribute content or functional code.

## Checklist (pick `Docs` or `Code`)

- [x] Docs contribution (philosophy, literature, content)
    - [x] **REQUIRED** I accept that my writing is submitted under the [ATTRIBUTION-NONCOMMERCIAL-SHAREALIKE 4.0 INTERNATIONAL](https://creativecommons.org/licenses/by-nc-sa/4.0/), which, briefly put, prohibits commercial re-use of the content, but allows sharing, remixing and building upon the material insofar as attribution is given (see [legal code](https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode.en)), and in this regard I understand that my writing is open to be modified, remixed and build upon by others in the sPhil community within the `systemphil/sphil` project in future perpetuity.
    - [x] **REQUIRED** I have followed the [formatting guidelines](https://github.com/systemphil/sphil/blob/dev/src/pages/contributing/formatting/basic-markdown.md).
    - [x] **REQUIRED** I have followed the [MLA citation style](https://owl.purdue.edu/owl/research_and_citation/mla_style/mla_formatting_and_style_guide/mla_formatting_and_style_guide.html).
    - [x] **REQUIRED** I have added or verified title and description metadata at the very top of the file. Example:

        ```md
        ---
        title: Hegel Guides
        description: Learn about the philosophy of G. W. F. Hegel
        ---
        ```

    - [ ] My article is a stub or I want to actively encourage contribution, I've added the Stub component to the bottom of my content but _before_ the bibliography:

        ```ts
        import Stub from "@/components/Stub";

        <Stub />
        ```

    - [x] **REQUIRED** I have added, verified or extended a bibliography. This is added before the authors section and follows MLA guidelines. Example:

        ```mdx
        ## Works Cited

        <div className="text-sm">
        - Hegel, Friedrich Georg Wilhelm. _The Science of Logic_. Translated by George di Giovanni, Cambridge University Press, 2010.
        - Kant, Immanuel. _The Critique of Pure Reason_. Edited by Paul Guyer and Allen W. Wood, Cambridge University Press, 1998.
        </div>
        ```

    - [x] I have signed the document with my name/username under either as `Authors`, `Editors` or `Contributors`. 
        > Use **Authors** if you have created and substantially added content.  
          Use **Editor** if you have made substantial edits or review.  
          Use **Contributor** if you have made minor edits, reviews or contributions.  
          If you've done multiple, pick the most weighted: Author > Editor > Contributor.  
          If you prefer to remain anonymous, that's fine too, but note that a record of your contributions based on your GitHub username will exist here in the codebase.

        ```md

        // Bibliography goes here

        ---

        **Authors**  
        Ahilleas Rokni (2024), Tom Bombadil (2025)

        **Contributors**  
        Filip Niklas (2024), Boromir (2025)
        ```

    - [ ] If the article makes use of footnotes, I have checked that `**Notes**` is exactly the very last item in the file.

- [ ] Code contribution ([Apache version 2 license](https://www.apache.org/licenses/LICENSE-2.0.txt))
    <details>

    All code apart of what is inside `src/pages/**` (excluding `/contributing/**`, `_app.mdx`, `_document.tsx`, `_meta.json`, `acknowledgements.mdx`, `index.mdx`, `privacy.mdx`, `team.mdx`, `terms.mdx`) is subject to Apache version 2 license. Basically, anything outside of content, literature, philosophy.

    </details>

### CODE REVIEW

File: src/pages/hegel/reference/mechanical-object/first-paragraph.mdx [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
...ticular`, and `individual`. It is these moments that are immediately identical to each other. In the `mechanical object`, the `universal` is immediately the `particular and the `individual`. In other words, the `universal` is not a universal that has the basic essence of a thing and that finds its essence instantiated in particular and individual objects. It is not, for example, like the universal concept of a chair that states that a chair must be ‘so and so’ and that serves as the essence of armchairs and swivel chairs, alike. It is not, as Hegel writes, a universal “in the sense of a community of properties” (Hegel, 1991, 711).  Rather, it is a universal that is immediately identical to the particular and the individual. In other words, the general concept of a chair is identical to all particular and individual chairs – there are not some chairs that are short and some that are long, or some chairs that offer good lumbar support and others that do not, rather, all chairs are identical to each other. The moments of the `mechanical object`, then, are treated as identical to each other, there is no distinction in conceiving of the `mechanical object` as universal or as individual.
```
Comment (Firgrep) [inline src/pages/hegel/reference/mechanical-object/first-paragraph.mdx:17]: There is a bug here. Probably an extra backtick skulking around somewhere or one missing. 

image

File: src/pages/hegel/reference/mechanical-object/first-paragraph.mdx [hunk 2]
```diff
@@ -1,17 +1,33 @@
 ---
 title: First Paragraph 
-description: Learn about the first paragraph of the Mechanical Object
+description: Learn about the First Paragraph of the Mechanical Object
 ---
```
Comment (Firgrep) [inline src/pages/hegel/reference/mechanical-object/first-paragraph.mdx:5]: Please add ## heading

File: src/pages/hegel/reference/mechanical-object/first-paragraph.mdx [hunk 3]
```diff
@@ -1,17 +1,33 @@
...
-What exactly are these moments of the “mechanical object” that have become an immediate identity? Hegel clarifies this in the following sentence:
+First, the mediation  of the syllogism, whatever that means in concrete terms, has been sublated . Strictly speaking, Hegel writes that it has been “balanced out” or “equilibrated” [the German being ausgeglichen]. It is because the mediation of the `syllogism` has been equilibrated that it was sublated.  As such, the mediation of the `syllogism` is not nullified but has been set aside by a more developed kind of relation – the relation of the `mechanical object` that is now an immediate identity [unmittelbare Identität]. The moments of the `mechanical object` are immediately identical to each other, and not mediated.
```
Comment (Firgrep) [inline src/pages/hegel/reference/mechanical-object/first-paragraph.mdx:10]: German/foreign words should be italicized. E.g. _ausgeglichen_ and _unmittelbare Identität_

File: src/pages/hegel/reference/mechanical-object/first-paragraph.mdx [hunk 4]
```diff
@@ -1,17 +1,33 @@
...
+What exactly are these moments of the `mechanical object` that have become an immediate identity? Hegel clarifies this in the following sentence:
```
Comment (Firgrep) [inline src/pages/hegel/reference/mechanical-object/first-paragraph.mdx:13]: Add empty line if you intended this to be a new paragraph.

File: src/pages/hegel/reference/mechanical-object/first-paragraph.mdx [hunk 5]
```diff
@@ -1,17 +1,33 @@
...
+“It is therefore in and for itself a universal - universality not in the sense of a community of properties, but a universality that pervades the particularity and in it is immediate individuality” (Hegel, 1991, 711).
```
Comment (Firgrep) [inline src/pages/hegel/reference/mechanical-object/first-paragraph.mdx:15]: Consider using block quote if you just have a paragraph that is a single sentence quote.

> Here is a block quote. Here you don't need quotation marks (Filip, 2024, 123).

File: src/pages/hegel/reference/mechanical-object/first-paragraph.mdx (hunk 1 above)
Comment (Firgrep) [reply src/pages/hegel/reference/mechanical-object/first-paragraph.mdx:17]: Also, single right quotation marks around "so and so", change these to apostrophes.

**Review by Firgrep — CHANGES_REQUESTED** [review]

File: src/pages/hegel/reference/mechanical-object/first-paragraph.mdx (hunk 1 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object/first-paragraph.mdx:17]: Done.

File: src/pages/hegel/reference/mechanical-object/first-paragraph.mdx (hunk 2 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object/first-paragraph.mdx:5]: Done.

File: src/pages/hegel/reference/mechanical-object/first-paragraph.mdx (hunk 3 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object/first-paragraph.mdx:10]: Done.

File: src/pages/hegel/reference/mechanical-object/first-paragraph.mdx (hunk 4 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object/first-paragraph.mdx:13]: Done.

File: src/pages/hegel/reference/mechanical-object/first-paragraph.mdx (hunk 5 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object/first-paragraph.mdx:15]: Done.

File: src/pages/hegel/reference/mechanical-object/first-paragraph.mdx [hunk 6]
```diff
@@ -1,17 +1,36 @@
...
-What exactly are these moments of the “mechanical object” that have become an immediate identity? Hegel clarifies this in the following sentence:
+>“The object is, as we have seen, the syllogism, whose mediation has been sublated and has therefore become an immediate identity” (Hegel, 1991, 711). Let’s unpack this initial thought, without trying to unpack the reference to `the syllogism`.
```
Comment (Firgrep) [inline src/pages/hegel/reference/mechanical-object/first-paragraph.mdx:10]: You left some of your own text inside of the block quote. 

image

You can also remove the quotation marks when using block quotes.

File: src/pages/hegel/reference/mechanical-object/first-paragraph.mdx [hunk 7]
```diff
@@ -1,17 +1,36 @@
...
 
+What exactly are these moments of the `mechanical object` that have become an immediate identity? Hegel clarifies this in the following sentence:
+
+>“It is therefore in and for itself a universal - universality not in the sense of a community of properties, but a universality that pervades the particularity and in it is immediate individuality” (Hegel, 1991, 711).
```
Comment (Firgrep) [inline src/pages/hegel/reference/mechanical-object/first-paragraph.mdx:18]: Remove quotation marks inside of block quote.

**Review by Firgrep — CHANGES_REQUESTED** [review]
A few tiny things left now

**Review by Firgrep — APPROVED** [review]

**Pull request merged**